src/components/MeasurementOverlay/EllipseMeasurementOverlay.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
+import Icon from 'components/Icon';
+import core from 'core';
+import getClassName from 'helpers/getClassName';
+import { mapAnnotationToKey, getDataWithKey } from '../../constants/map';
+
+function EllipseMeasurementOverlay(props) {
+  const { t, annotation, isOpen } = props;
+  const annotationKey = mapAnnotationToKey(annotation);
+  const { icon } = getDataWithKey(annotationKey);
+  const scale = annotation.Scale;
+  const precision = annotation.Precision;
+  const unit = annotation.Scale[1][1];
+  const className = getClassName('Overlay MeasurementOverlay', { isOpen });
+
+  const renderScaleRatio = () => `${scale[0][0]} ${scale[0][1]} = ${scale[1][0]} ${scale[1][1]}`;
+
+  const computeRadius = () => {
+    const decimalPlaces = 2;
```

Comment:
Do we want to use the logic in `getNumberOfDecimalPlaces` instead of having this to be fixed at 2?